changelogs/fragments/proxy-able-to-use-interface.yaml

```
@@ -0,0 +1,3 @@
+bugfixes:
+    - "The 'interface' option was missing when creating an Proxy via the API."
+    - "Added new property 'zabbix_proxy_ip' to determine ip for host running the Zabbix Proxy."
```

Can we always add role/module name to the changelog file so it is easier to read in the final changelog?

```suggestion
    - zabbix_agent - "The 'interface' option was missing when creating an Proxy via the API."
    - zabbix_agent - "Added new property 'zabbix_proxy_ip' to determine ip for host running the Zabbix Proxy."
```

---

Yes, done! 👍